testing/opencolorio/APKBUILD

```diff
@@ -0,0 +1,69 @@
+# Contributor: Mark Riedesel <mark@klowner.com>
+# Maintainer:
+pkgname=opencolorio
+pkgver=1.0.9
+pkgrel=0
+pkgdesc="A color management framework for visual effects and animation"
+url="http://opencolorio.org"
+arch="all"
+license="BSD"
+depends=""
+makedepends="cmake python2-dev python3-dev freeglut-dev glew-dev"
+install=""
+subpackages="$pkgname-python2:py2 $pkgname-python3:py3 $pkgname-dev $pkgname-utils"
+source="$pkgname-$pkgver.tar.gz::http://github.com/imageworks/OpenColorIO/archive/v$pkgver.tar.gz"
+builddir="$srcdir"/OpenColorIO-$pkgver
+
+build() {
+	cd "$builddir"
+
+	for pyver in python3 python2; do
```

Comment:
This variable should be declared as local (`local pyver`), to not pollute global namespace of the script.